ui-tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
+https://jupyterlite-pyodide-kernel--183.org.readthedocs.build/en/183/_static/jupyterlite_pyodide_kernel-0.6.0a5-py3-none-any.whl
```

Comment:
Maybe we can also update the version of `jupyterlite-pyodide-kernel` used in the demo site here with this link too:

https://github.com/jupyterlite/jupyterlite/blob/e17054905a26304d5b8f643af3e34c81d91e590c/examples/jupyter_lite_config.json#L5

So it's easier on the RTD preview for this PR.